--- a/web/gui/main.js
+++ b/web/gui/main.js
@@ -148,7 +148,7 @@
                     urlOptions[numeric[len]] = parseInt(urlOptions[numeric[len]]);
                 }
                 catch (e) {
-                    console.log('failed to parse URL hash parameter ' + numeric[len]);
+                    console.log('无法解析URL哈希参数 ' + numeric[len]);
                     urlOptions[numeric[len]] = 0;
                 }
             }
@@ -333,7 +333,7 @@
         if (localStorageTest() === true) {
             ret = localStorage.getItem(name);
         } else {
-            console.log('localStorage is not available');
+            console.log('本地存储不可用');
         }
     }
     catch (error) {
@@ -575,7 +575,7 @@
         let ret = loadLocalStorage("registryCallback");
         if (ret) {
             machinesArray = JSON.parse(ret);
-            console.log("failed to contact the registry - loaded registry data from browser local storage");
+            console.log("无法联系从浏览器本地存储加载的注册表数据");
         }
     }
 
@@ -664,17 +664,17 @@
         }
 
         html += `<hr />`;
-        html += `<div class="info-item">Demo netdata nodes</div>`;
+        html += `<div class="info-item">演示网络数据节点</div>`;
 
         const demoServers = [
-            { url: "//london.netdata.rocks/default.html", title: "UK - London (DigitalOcean.com)" },
-            { url: "//newyork.netdata.rocks/default.html", title: "US - New York (DigitalOcean.com)" },
-            { url: "//sanfrancisco.netdata.rocks/default.html", title: "US - San Francisco (DigitalOcean.com)" },
-            { url: "//atlanta.netdata.rocks/default.html", title: "US - Atlanta (CDN77.com)" },
-            { url: "//frankfurt.netdata.rocks/default.html", title: "Germany - Frankfurt (DigitalOcean.com)" },
-            { url: "//toronto.netdata.rocks/default.html", title: "Canada - Toronto (DigitalOcean.com)" },
-            { url: "//singapore.netdata.rocks/default.html", title: "Japan - Singapore (DigitalOcean.com)" },
-            { url: "//bangalore.netdata.rocks/default.html", title: "India - Bangalore (DigitalOcean.com)" },
+            { url: "//london.netdata.rocks/default.html", title: "英国 - 伦敦 (DigitalOcean.com)" },
+            { url: "//newyork.netdata.rocks/default.html", title: "美国 - 纽约 (DigitalOcean.com)" },
+            { url: "//sanfrancisco.netdata.rocks/default.html", title: "美国 - 旧金山 (DigitalOcean.com)" },
+            { url: "//atlanta.netdata.rocks/default.html", title: "美国 - 亚特兰大 (CDN77.com)" },
+            { url: "//frankfurt.netdata.rocks/default.html", title: "德国 - 法兰克福 (DigitalOcean.com)" },
+            { url: "//toronto.netdata.rocks/default.html", title: "加拿大 - 多伦多 (DigitalOcean.com)" },
+            { url: "//singapore.netdata.rocks/default.html", title: "新加坡 - 新加坡 (DigitalOcean.com)" },
+            { url: "//bangalore.netdata.rocks/default.html", title: "印度 - 班加罗尔 (DigitalOcean.com)" },
 
         ]
 
@@ -701,7 +701,7 @@
 function clearMyNetdataMenu() {
     setMyNetdataMenu(`<div class="agent-item" style="white-space: nowrap">
         <i class="fas fa-hourglass-half"></i>
-        Loading, please wait...
+        加载中请稍候...
         <div></div>
     </div>`);
 }
@@ -709,7 +709,7 @@
 function errorMyNetdataMenu() {
     setMyNetdataMenu(`<div class="agent-item" style="padding: 0 8px">
         <i class="fas fa-exclamation-triangle" style="color: red"></i>
-        Cannot load known Netdata agents from Netdata Cloud! Please make sure you have the latest version of Netdata.
+        无法从Netdata Cloud加载已知的Netdata代理！请确保您具有最新版本的Netdata。
     </div>`);
 }
 
@@ -733,7 +733,7 @@
     el.classList.add(`theme-${netdataTheme}`);
 
     if (machinesArray == registryAgents) {
-        console.log("Rendering my-netdata menu from registry");
+        console.log("从注册表呈现我的netdata菜单");
     } else {
         console.log("Rendering my-netdata menu from netdata.cloud", machinesArray);
     }
@@ -935,7 +935,7 @@
                 gotoServerValidateRemaining--;
                 if (gotoServerValidateRemaining <= 0) {
                     gotoServerMiddleClick = false;
-                    document.getElementById('gotoServerResponse').innerHTML = '<b>Sorry! I cannot find any operational URL for this server</b>';
+                    document.getElementById('gotoServerResponse').innerHTML = '<b>抱歉! 我找不到该服务器的任何可操作网址</b>';
                 }
             }
         });
@@ -967,7 +967,7 @@
         // we consult the registry to get additional URLs.
         setTimeout(function () {
             if (gotoServerStop === false) {
-                document.getElementById('gotoServerResponse').innerHTML = '<b>Added all the known URLs for this machine.</b>';
+                document.getElementById('gotoServerResponse').innerHTML = '<b>添加了此计算机的所有已知URL。</b>';
                 NETDATA.registry.search(guid, function (data) {
                     // console.log(data);
                     len = data.urls.length;
@@ -1017,7 +1017,7 @@
                 $('#switchRegistryModal').modal('hide');
                 NETDATA.registry.init();
             } else {
-                document.getElementById('switchRegistryResponse').innerHTML = "<b>Sorry! The registry rejected your request.</b>";
+                document.getElementById('switchRegistryResponse').innerHTML = "<b>抱歉! 注册表拒绝了您的请求。</b>";
             }
         });
     } else {
@@ -1050,12 +1050,12 @@
             deleteCloudAgentURL(deleteRegistryGuid, deleteRegistryUrl)
                 .then((count) => {
                     if (!count) {
-                        responseEl.innerHTML = "<b>Sorry, this command was rejected by netdata.cloud!</b>";
+                        responseEl.innerHTML = "<b>抱歉，此命令已被netdata.cloud拒绝！</b>";
                         return;
                     }
                     NETDATA.registry.delete(deleteRegistryUrl, function (result) {
                         if (result === null) {
-                            console.log("Received error from registry", result);
+                            console.log("从注册表收到错误", result);
                         }
 
                         deleteRegistryUrl = null;
@@ -1070,7 +1070,7 @@
                     $('#deleteRegistryModal').modal('hide');
                     NETDATA.registry.init();
                 } else {
-                    responseEl.innerHTML = "<b>Sorry, this command was rejected by the registry server!</b>";
+                    responseEl.innerHTML = "<b>抱歉，此命令已被注册表服务器拒绝！</b>";
                 }
             });
         }
@@ -1497,8 +1497,8 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.io"'
             + ' data-dimensions="in"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="Disk Read"'
-            + ' data-width="11%"'
+            + ' data-title="存储读取"'
+            + ' data-width="14%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
             + ' data-points="' + duration.toString() + '"'
@@ -1508,8 +1508,8 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.io"'
             + ' data-dimensions="out"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="Disk Write"'
-            + ' data-width="11%"'
+            + ' data-title="存储写入"'
+            + ' data-width="14%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
             + ' data-points="' + duration.toString() + '"'
@@ -1520,8 +1520,8 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.pgpgio"'
             + ' data-dimensions="in"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="Disk Read"'
-            + ' data-width="11%"'
+            + ' data-title="磁盘读取"'
+            + ' data-width="14%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
             + ' data-points="' + duration.toString() + '"'
@@ -1531,8 +1531,8 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.pgpgio"'
             + ' data-dimensions="out"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="Disk Write"'
-            + ' data-width="11%"'
+            + ' data-title="磁盘写入"'
+            + ' data-width="14%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
             + ' data-points="' + duration.toString() + '"'
@@ -1546,7 +1546,7 @@
             + ' data-title="CPU"'
             + ' data-units="%"'
             + ' data-gauge-max-value="100"'
-            + ' data-width="20%"'
+            + ' data-width="25%"'
             + ' data-after="-' + duration.toString() + '"'
             + ' data-points="' + duration.toString() + '"'
             + ' data-colors="' + NETDATA.colors[12] + '"'
@@ -1557,7 +1557,7 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.net"'
             + ' data-dimensions="received"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="Net Inbound"'
+            + ' data-title="下载速度"'
             + ' data-width="11%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
@@ -1568,7 +1568,7 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.net"'
             + ' data-dimensions="sent"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="Net Outbound"'
+            + ' data-title="上传速度"'
             + ' data-width="11%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
@@ -1580,7 +1580,7 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.ip"'
             + ' data-dimensions="received"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="IP Inbound"'
+            + ' data-title="IP下载"'
             + ' data-width="11%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
@@ -1591,7 +1591,7 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.ip"'
             + ' data-dimensions="sent"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="IP Outbound"'
+            + ' data-title="IP上传"'
             + ' data-width="11%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
@@ -1603,7 +1603,7 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.ipv4"'
             + ' data-dimensions="received"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="IPv4 Inbound"'
+            + ' data-title="IPv4下载"'
             + ' data-width="11%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
@@ -1614,7 +1614,7 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.ipv4"'
             + ' data-dimensions="sent"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="IPv4 Outbound"'
+            + ' data-title="IPv4上传"'
             + ' data-width="11%"'
             + ' data-before="0"'
             + ' data-after="-' + duration.toString() + '"'
@@ -1626,7 +1626,7 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.ipv6"'
             + ' data-dimensions="received"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="IPv6 Inbound"'
+            + ' data-title="IPv6下载"'
             + ' data-units="kbps"'
             + ' data-width="11%"'
             + ' data-before="0"'
@@ -1638,7 +1638,7 @@
         head += '<div class="netdata-container" style="margin-right: 10px;" data-netdata="system.ipv6"'
             + ' data-dimensions="sent"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="IPv6 Outbound"'
+            + ' data-title="IPv6上传"'
             + ' data-units="kbps"'
             + ' data-width="11%"'
             + ' data-before="0"'
@@ -1653,7 +1653,7 @@
             + ' data-dimensions="used|buffers|active|wired"' // active and wired are FreeBSD stats
             + ' data-append-options="percentage"'
             + ' data-chart-library="easypiechart"'
-            + ' data-title="Used RAM"'
+            + ' data-title="内存"'
             + ' data-units="%"'
             + ' data-easypiechart-max-value="100"'
             + ' data-width="9%"'
@@ -1804,20 +1804,21 @@
 
     const isMemoryModeDbEngine = data.memory_mode === "dbengine";
 
-    sidebar += '<li class="" style="padding-top:15px;"><a href="https://docs.netdata.cloud/collectors/quickstart/" target="_blank"><i class="fas fa-plus"></i> Add more charts</a></li>';
-    sidebar += '<li class=""><a href="https://docs.netdata.cloud/health/quickstart/" target="_blank"><i class="fas fa-plus"></i> Add more alarms</a></li>';
-    sidebar += '<li class="" style="margin:20px;color:#666;"><small>Every ' +
-      ((data.update_every === 1) ? 'second' : data.update_every.toString() + ' seconds') + ', ' +
-      'Netdata collects <strong>' + data.dimensions_count.toLocaleString() + '</strong> metrics on ' +
-      data.hostname.toString() + ', presents them in <strong>' +
-      data.charts_count.toLocaleString() + '</strong> charts' +
+    sidebar += '<li class="" style="padding-top:15px;"><a href="https://docs.netdata.cloud/collectors/quickstart/" target="_blank"><i class="fas fa-plus"></i>添加更多图表</a></li>';
+    sidebar += '<li class=""><a href="https://docs.netdata.cloud/health/quickstart/" target="_blank"><i class="fas fa-plus"></i>添加更多图表</a></li>';
+    sidebar += '<li class="" style="margin:20px;color:#666;"><small>Netdata以每' +
+      ((data.update_every === 1) ? '秒' : data.update_every.toString() + ' seconds') +
+      '在<strong>' + 
+      data.hostname.toString() + '</strong>上收集' +
+      data.dimensions_count.toLocaleString()  + '个指标, 以<strong>' +
+      data.charts_count.toLocaleString() + '</strong>个图表的形式显示' +
       (isMemoryModeDbEngine ? '' : ',') + // oxford comma
-      ' and monitors them with <strong>' +
-      data.alarms_count.toLocaleString() + '</strong> alarms.';
+      ' 并用<strong>' +
+      data.alarms_count.toLocaleString() + '</strong>警报监视它们。';
 
     if (!isMemoryModeDbEngine) {
-        sidebar += '<br />&nbsp;<br />Get more history by ' +
-          '<a href="https://docs.netdata.cloud/docs/configuration-guide/#increase-the-metrics-retention-period" target=_blank>configuring Netdata\'s <strong>history</strong></a> or using the <a href="https://docs.netdata.cloud/database/engine/" target=_blank>DB engine.</a>';
+        sidebar += '<br />&nbsp;<br />获取更多历史' +
+          '<a href="https://docs.netdata.cloud/docs/configuration-guide/#increase-the-metrics-retention-period" target=_blank>Netdata配置</a>或者使用<a href="https://docs.netdata.cloud/database/engine/" target=_blank>DB数据库引擎。</a>';
     }
 
     sidebar += '<br/>&nbsp;<br/><strong>netdata</strong><br/>' + data.version.toString() + '</small></li>';
@@ -2022,9 +2023,9 @@
 // ----------------------------------------------------------------------------
 
 function alarmsUpdateModal() {
-    var active = '<h3>Raised Alarms</h3><table class="table">';
-    var all = '<h3>All Running Alarms</h3><div class="panel-group" id="alarms_all_accordion" role="tablist" aria-multiselectable="true">';
-    var footer = '<hr/><a href="https://github.com/netdata/netdata/tree/master/web/api/badges#netdata-badges" target="_blank">netdata badges</a> refresh automatically. Their color indicates the state of the alarm: <span style="color: #e05d44"><b>&nbsp;red&nbsp;</b></span> is critical, <span style="color:#fe7d37"><b>&nbsp;orange&nbsp;</b></span> is warning, <span style="color: #4c1"><b>&nbsp;bright green&nbsp;</b></span> is ok, <span style="color: #9f9f9f"><b>&nbsp;light grey&nbsp;</b></span> is undefined (i.e. no data or no status), <span style="color: #000"><b>&nbsp;black&nbsp;</b></span> is not initialized. You can copy and paste their URLs to embed them in any web page.<br/>netdata can send notifications for these alarms. Check <a href="https://github.com/netdata/netdata/blob/master/health/notifications/health_alarm_notify.conf" target="_blank">this configuration file</a> for more information.';
+    var active = '<h3>错误信息</h3><table class="table">';
+    var all = '<h3>所有运行的错误信息</h3><div class="panel-group" id="alarms_all_accordion" role="tablist" aria-multiselectable="true">';
+    var footer = '<hr/><a href="https://github.com/netdata/netdata/tree/master/web/api/badges#netdata-badges" target="_blank">netdata badges</a>会自动重新整理。不同颜色分表代表的错误状态：<span style="color: #e05d44"><b>&nbsp;红色&nbsp;</b></span>表示严重，<span style="color:#fe7d37"><b>&nbsp;橘色&nbsp;</b></span>表示警告，<span style="color: #4c1"><b>&nbsp;绿色&nbsp;</b></span>表示良好，<span style="color: #9f9f9f"><b>&nbsp;灰色&nbsp;</b></span>表示未定义(例如无资料或无状态)，<span style="color: #000"><b>&nbsp;黑色&nbsp;</b></span> 表示尚未初始化。您可以复制这里的网址并将它们嵌入到任一个网页。<br/>netdata 能够发送这些警报通知。请参阅 <a href="https://github.com/netdata/netdata/blob/master/health/notifications/health_alarm_notify.conf">这个文档</a>了解更多信息。';
 
     loadClipboard(function () {
     });
@@ -2038,7 +2039,7 @@
             document.getElementById('alarms_active').innerHTML =
                 document.getElementById('alarms_all').innerHTML =
                 document.getElementById('alarms_log').innerHTML =
-                'failed to load alarm data!';
+                '无法加载错误信息数据！';
             return;
         }
 
@@ -2218,7 +2219,7 @@
 
             // not found - this should never happen!
             if (typeof chart === 'undefined') {
-                console.log('WARNING: alarm ' + x + ' is linked to chart ' + alarm.chart + ', which is not found in the list of chart got from the server.');
+                console.log('WARNING: alarm ' + x + ' 链接到图表 ' + alarm.chart + '，在从服务器获取的图表列表中找不到。');
                 chart = { priority: 9999999 };
             }
             else if (typeof chart.menu !== 'undefined' && typeof chart.submenu !== 'undefined')
@@ -2301,13 +2302,13 @@
         all += "</div>";
 
         if (!count_active) {
-            active += '<div style="width:100%; height: 100px; text-align: center;"><span style="font-size: 50px;"><i class="fas fa-thumbs-up"></i></span><br/>Everything is normal. No raised alarms.</div>';
+            active += '<div style="width:100%; height: 100px; text-align: center;"><span style="font-size: 50px;"><i class="fas fa-thumbs-up"></i></span><br/>一切正常，没有错误信息。</div>';
         } else {
             active += footer;
         }
 
         if (!count_all) {
-            all += "<h4>No alarms are running in this system.</h4>";
+            all += "<h4>系统中没有错误信息。</h4>";
         } else {
             all += footer;
         }
@@ -2333,7 +2334,7 @@
             $('#alarm_all_' + id.toString()).html('');
         });
 
-        document.getElementById('alarms_log').innerHTML = '<h3>Alarm Log</h3><table id="alarms_log_table"></table>';
+        document.getElementById('alarms_log').innerHTML = '<h3>日志记录</h3><table id="alarms_log_table"></table>';
 
         loadBootstrapTable(function () {
             $('#alarms_log_table').bootstrapTable({
@@ -2391,9 +2392,9 @@
                 columns: [
                     {
                         field: 'when',
-                        title: 'Event Date',
+                        title: '事件日期',
                         valign: 'middle',
-                        titleTooltip: 'The date and time the even took place',
+                        titleTooltip: '发生日期和时间',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2405,17 +2406,17 @@
                     },
                     {
                         field: 'hostname',
-                        title: 'Host',
+                        title: '主机',
                         valign: 'middle',
-                        titleTooltip: 'The host that generated this event',
+                        titleTooltip: '产生此事件的主机',
                         align: 'center',
                         visible: false,
                         sortable: true
                     },
                     {
                         field: 'unique_id',
-                        title: 'Unique ID',
-                        titleTooltip: 'The host unique ID for this event',
+                        title: '唯一ID',
+                        titleTooltip: '此事件的主机唯一ID',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2428,8 +2429,8 @@
                     },
                     {
                         field: 'alarm_id',
-                        title: 'Alarm ID',
-                        titleTooltip: 'The ID of the alarm that generated this event',
+                        title: '警报ID',
+                        titleTooltip: '生成此事件的警报的ID',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2442,8 +2443,8 @@
                     },
                     {
                         field: 'alarm_event_id',
-                        title: 'Alarm Event ID',
-                        titleTooltip: 'The incremental ID of this event for the given alarm',
+                        title: '警报事件ID',
+                        titleTooltip: '给定警报的此事件的增量ID',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2456,8 +2457,8 @@
                     },
                     {
                         field: 'chart',
-                        title: 'Chart',
-                        titleTooltip: 'The chart the alarm is attached to',
+                        title: '图表',
+                        titleTooltip: '警报所附图表',
                         align: 'center',
                         valign: 'middle',
                         switchable: false,
@@ -2465,8 +2466,8 @@
                     },
                     {
                         field: 'family',
-                        title: 'Family',
-                        titleTooltip: 'The family of the chart the alarm is attached to',
+                        title: '家庭',
+                        titleTooltip: '警报所附图表的系列',
                         align: 'center',
                         valign: 'middle',
                         visible: false,
@@ -2474,8 +2475,8 @@
                     },
                     {
                         field: 'name',
-                        title: 'Alarm',
-                        titleTooltip: 'The alarm name that generated this event',
+                        title: '错误信息',
+                        titleTooltip: '生成此事件的警报名称',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2489,7 +2490,7 @@
                     {
                         field: 'value_string',
                         title: 'Friendly Value',
-                        titleTooltip: 'The value of the alarm, that triggered this event',
+                        titleTooltip: '触发此事件的警报的值',
                         align: 'right',
                         valign: 'middle',
                         sortable: true
@@ -2497,7 +2498,7 @@
                     {
                         field: 'old_value_string',
                         title: 'Friendly Old Value',
-                        titleTooltip: 'The value of the alarm, just before this event',
+                        titleTooltip: '此事件之前的警报值',
                         align: 'right',
                         valign: 'middle',
                         visible: false,
@@ -2506,7 +2507,7 @@
                     {
                         field: 'old_value',
                         title: 'Old Value',
-                        titleTooltip: 'The value of the alarm, just before this event',
+                        titleTooltip: '此事件之前的警报值',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2520,7 +2521,7 @@
                     {
                         field: 'value',
                         title: 'Value',
-                        titleTooltip: 'The value of the alarm, that triggered this event',
+                        titleTooltip: '触发此事件的警报的值',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2533,8 +2534,8 @@
                     },
                     {
                         field: 'units',
-                        title: 'Units',
-                        titleTooltip: 'The units of the value of the alarm',
+                        title: '单位',
+                        titleTooltip: '报警值的单位',
                         align: 'left',
                         valign: 'middle',
                         visible: false,
@@ -2542,8 +2543,8 @@
                     },
                     {
                         field: 'old_status',
-                        title: 'Old Status',
-                        titleTooltip: 'The status of the alarm, just before this event',
+                        title: '先前状态',
+                        titleTooltip: '此事件之前的警报状态',
                         align: 'center',
                         valign: 'middle',
                         visible: false,
@@ -2551,8 +2552,8 @@
                     },
                     {
                         field: 'status',
-                        title: 'Status',
-                        titleTooltip: 'The status of the alarm, that was set due to this event',
+                        title: '状态',
+                        titleTooltip: '由于该事件而设置的警报状态',
                         align: 'center',
                         valign: 'middle',
                         switchable: false,
@@ -2560,8 +2561,8 @@
                     },
                     {
                         field: 'duration',
-                        title: 'Last Duration',
-                        titleTooltip: 'The duration the alarm was at its previous state, just before this event',
+                        title: '最后持续时间',
+                        titleTooltip: '警报在其之前状态（在此事件之前）的持续时间',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2574,8 +2575,8 @@
                     },
                     {
                         field: 'non_clear_duration',
-                        title: 'Raised Duration',
-                        titleTooltip: 'The duration the alarm was raised, just before this event',
+                        title: '持续时间',
+                        titleTooltip: '警报发出的持续时间，就在此事件之前',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2588,8 +2589,8 @@
                     },
                     {
                         field: 'recipient',
-                        title: 'Recipient',
-                        titleTooltip: 'The recipient of this event',
+                        title: '接受者',
+                        titleTooltip: '此活动的接收者',
                         align: 'center',
                         valign: 'middle',
                         visible: false,
@@ -2597,8 +2598,8 @@
                     },
                     {
                         field: 'processed',
-                        title: 'Processed Status',
-                        titleTooltip: 'True when this event is processed',
+                        title: '处理状态',
+                        titleTooltip: '处理此事件时为True',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2616,8 +2617,8 @@
                     },
                     {
                         field: 'updated',
-                        title: 'Updated Status',
-                        titleTooltip: 'True when this event has been updated by another event',
+                        title: '更新状态',
+                        titleTooltip: '如果此事件已由其他事件更新，则为True',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2635,8 +2636,8 @@
                     },
                     {
                         field: 'updated_by_id',
-                        title: 'Updated By ID',
-                        titleTooltip: 'The unique ID of the event that obsoleted this one',
+                        title: '按ID更新',
+                        titleTooltip: '使该事件失效的事件的唯一ID',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2649,8 +2650,8 @@
                     },
                     {
                         field: 'updates_id',
-                        title: 'Updates ID',
-                        titleTooltip: 'The unique ID of the event obsoleted because of this event',
+                        title: '更新ID',
+                        titleTooltip: '由于此事件，该事件的唯一ID已过时',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2663,8 +2664,8 @@
                     },
                     {
                         field: 'exec',
-                        title: 'Script',
-                        titleTooltip: 'The script to handle the event notification',
+                        title: '脚本',
+                        titleTooltip: '处理事件通知的脚本',
                         align: 'center',
                         valign: 'middle',
                         visible: false,
@@ -2672,8 +2673,8 @@
                     },
                     {
                         field: 'exec_run',
-                        title: 'Script Run At',
-                        titleTooltip: 'The date and time the script has been ran',
+                        title: '脚本运行于',
+                        titleTooltip: '脚本运行的日期和时间',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2686,8 +2687,8 @@
                     },
                     {
                         field: 'exec_code',
-                        title: 'Script Return Value',
-                        titleTooltip: 'The return code of the script',
+                        title: '脚本返回值',
+                        titleTooltip: '脚本的返回代码',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2705,8 +2706,8 @@
                     },
                     {
                         field: 'delay',
-                        title: 'Script Delay',
-                        titleTooltip: 'The hysteresis of the notification',
+                        title: '脚本延迟',
+                        titleTooltip: '通知的滞后性',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2720,8 +2721,8 @@
                     },
                     {
                         field: 'delay_up_to_timestamp',
-                        title: 'Script Delay Run At',
-                        titleTooltip: 'The date and time the script should be run, after hysteresis',
+                        title: '脚本延迟运行时间',
+                        titleTooltip: '在滞后之后，脚本应该运行的日期和时间',
                         formatter: function (value, row, index) {
                             void (row);
                             void (index);
@@ -2734,8 +2735,8 @@
                     },
                     {
                         field: 'info',
-                        title: 'Description',
-                        titleTooltip: 'A short description of the alarm',
+                        title: '说明',
+                        titleTooltip: '警报的简短描述',
                         align: 'center',
                         valign: 'middle',
                         visible: false,
@@ -2743,8 +2744,8 @@
                     },
                     {
                         field: 'source',
-                        title: 'Alarm Source',
-                        titleTooltip: 'The source of configuration of the alarm',
+                        title: '警报来源',
+                        titleTooltip: '警报的配置来源',
                         align: 'center',
                         valign: 'middle',
                         visible: false,
@@ -2803,7 +2804,7 @@
         }
 
         // update the dashboard title
-        document.title = options.hostname + ' netdata dashboard';
+        document.title = options.hostname + ' netdata仪表板';
 
         // close the splash screen
         $("#loadOverlay").css("display", "none");
@@ -2946,7 +2947,7 @@
 }
 
 function getGithubLatestVersion(callback) {
-    versionLog('Downloading latest version id from github...');
+    versionLog('正在从github下载最新版本信息...');
 
     $.ajax({
         url: 'https://api.github.com/repos/netdata/netdata/releases/latest',
@@ -2955,11 +2956,11 @@
     })
         .done(function (data) {
             data = data.tag_name.replace(/(\r\n|\n|\r| |\t)/gm, "");
-            versionLog('Latest stable version from github is ' + data);
+            versionLog('从github取得最新版本是' + data);
             callback(data);
         })
         .fail(function () {
-            versionLog('Failed to download the latest stable version id from github!');
+            versionLog('无法从github下载最新的稳定版本ID！');
             callback(null);
         });
 }
@@ -3008,7 +3009,7 @@
 }
 
 function notifyForUpdate(force) {
-    versionLog('<p>checking for updates...</p>');
+    versionLog('<p>正在检查更新...</p>');
 
     var now = Date.now();
 
@@ -3032,17 +3033,17 @@
 
         if (sha1 === null) {
             save = false;
-            versionLog('<p><big>Failed to get your netdata version!</big></p><p>You can always get the latest netdata from <a href="https://github.com/netdata/netdata" target="_blank">its github page</a>.</p>');
+            versionLog('<p><big>取得您的 netdata 版本失败！</big></p><p> 您始终可以从以下位置获取最新的netdata <a href="https://github.com/netdata/netdata" target="_blank">它的github页面</a>.</p>');
         } else if (sha2 === null) {
             save = false;
-            versionLog('<p><big>Failed to get the latest netdata version.</big></p><p>You can always get the latest netdata from <a href="https://github.com/netdata/netdata" target="_blank">its github page</a>.</p>');
+            versionLog('<p><big>从github取得netdata最新版本失败。</big></p><p>您也可以从<a href="https://github.com/netdata/netdata" target="_blank">github</a>取得最新netdata版本。</p>');
         } else if (versionsMatch(sha1, sha2)) {
             save = true;
-            versionLog('<p><big>You already have the latest netdata!</big></p><p>No update yet?<br/>We probably need some motivation to keep going on!</p><p>If you haven\'t already, <a href="https://github.com/netdata/netdata" target="_blank">give netdata a <b><i class="fas fa-star"></i></b> at its github page</a>.</p>');
+            versionLog('<p><big>您已经是最新版本的 netdata！</big></p><p>还没有更新？<br/>或许，我们还需要一些动力继续前进！</p><p>如果您还没有做好更新的准备，请您 <a href="https://github.com/netdata/netdata" target="_blank">到 github 给 netdata <b><i class="fas fa-star"></i></b></a>。</p>');
         } else {
             save = true;
             var compare = 'https://docs.netdata.cloud/changelog/';
-            versionLog('<p><big><strong>New version of netdata available!</strong></big></p><p>Latest version: <b><code>' + sha2 + '</code></b></p><p><a href="' + compare + '" target="_blank">Click here for the changes log</a> and<br/><a href="https://github.com/netdata/netdata/tree/master/packaging/installer/UPDATE.md" target="_blank">click here for directions on updating</a> your netdata installation.</p><p>We suggest to review the changes log for new features you may be interested, or important bug fixes you may need.<br/>Keeping your netdata updated is generally a good idea.</p>');
+            versionLog('<p><big><strong>现在有新的netdata版本可用！</strong></big></p><p>可用的最新版本是：<b><code>' + sha2 + '</code></b></p><p><a href="' + compare + '" target="_blank">单击此处查看更改日志</a><br/><a href="https://github.com/netdata/netdata/tree/master/packaging/installer/UPDATE.md" target="_blank">单击此处获取更新说明</a></p><p>我们建议查看更改日志，以了解您可能感兴趣的新功能，或您可能需要的重要错误修复。<br/>保持您的网络数据更新通常是一个好主意。</p>');
 
             document.getElementById('update_badge').innerHTML = '!';
         }
@@ -3285,13 +3286,13 @@
 
     if (tmpSnapshotData === null) {
         loadSnapshotPreflightEmpty();
-        loadSnapshotModalLog('danger', 'no data have been loaded');
+        loadSnapshotModalLog('danger', '尚未加载任何数据');
         return;
     }
 
     loadPako(function () {
         loadLzString(function () {
-            loadSnapshotModalLog('info', 'Please wait, activating snapshot...');
+            loadSnapshotModalLog('info', '请稍候，正在激活快照...');
             $('#loadSnapshotModal').modal('hide');
 
             netdataShowAlarms = false;
@@ -3400,7 +3401,7 @@
             document.getElementById('loadSnapshotInfo').innerHTML = 'version: <b>' + result.snapshot_version.toString() + '</b>, includes <b>' + result.charts_ok.toString() + '</b> unique chart data queries ' + ((result.charts_failed > 0) ? ('<b>' + result.charts_failed.toString() + '</b> failed') : '').toString() + ', compressed with <code>' + result.compression.toString() + '</code>, data size ' + (Math.round(result.data_size * 100 / 1024 / 1024) / 100).toString() + ' MB';
             document.getElementById('loadSnapshotTimeRange').innerHTML = '<b>' + NETDATA.dateTime.localeDateString(date_after) + ' ' + NETDATA.dateTime.localeTimeString(date_after) + '</b> to <b>' + NETDATA.dateTime.localeDateString(date_before) + ' ' + NETDATA.dateTime.localeTimeString(date_before) + '</b>';
             document.getElementById('loadSnapshotComments').innerHTML = ((result.comments) ? result.comments : '').toString();
-            loadSnapshotModalLog('success', 'File loaded, click <b>Import</b> to render it!');
+            loadSnapshotModalLog('success', '文件已加载，请点击<b>导入</b>进行渲染！');
             $('#loadSnapshotImport').removeClass('disabled');
 
             tmpSnapshotData = result;
@@ -3444,7 +3445,7 @@
                     loadSnapshotPreflightFile(e.originalEvent.dataTransfer.files.item(0));
                 } else {
                     loadSnapshotPreflightEmpty();
-                    loadSnapshotModalLog('danger', 'No file selected');
+                    loadSnapshotModalLog('danger', '未选择文件');
                 }
             });
     }
@@ -3454,11 +3455,11 @@
     var files = document.getElementById('loadSnapshotSelectFiles').files;
     if (files.length <= 0) {
         loadSnapshotPreflightEmpty();
-        loadSnapshotModalLog('danger', 'No file selected');
+        loadSnapshotModalLog('danger', '未选择文件');
         return;
     }
 
-    loadSnapshotModalLog('info', 'Loading file...');
+    loadSnapshotModalLog('info', '正在载入档案...');
 
     loadSnapshotPreflightFile(files.item(0));
 }
@@ -3495,7 +3496,7 @@
 function saveSnapshotModalShowExpectedSize() {
     var points = Math.round(saveSnapshotViewDuration / saveSnapshotSelectedSecondsPerPoint);
     var priority = 'info';
-    var msg = 'A moderate snapshot.';
+    var msg = '中等快照。';
 
     var sizemb = Math.round(
         (options.data.charts_count * snapshotOptions.bytes_per_chart
@@ -3509,18 +3510,18 @@
 
     if (sizemb < 10) {
         priority = 'success';
-        msg = 'A nice small snapshot!';
+        msg = '一个不错的小快照！';
     }
     if (sizemb > 50) {
         priority = 'warning';
-        msg = 'Will stress your browser...';
+        msg = '会给您的浏览器造成压力...';
     }
     if (sizemb > 100) {
         priority = 'danger';
-        msg = 'Hm... good luck...';
+        msg = '嗯...祝你好运...';
     }
 
-    saveSnapshotModalLog(priority, 'The snapshot will have ' + points.toString() + ' points per dimension. Expected size on disk ' + sizemb + ' MB, at browser memory ' + memmb + ' MB.<br/>' + msg);
+    saveSnapshotModalLog(priority, '快照每个维度有 ' + points.toString() + ' 个点。磁盘上的预期保存大小为 ' + sizemb + ' MB, 浏览器内存为 ' + memmb + ' MB。<br/>' + msg);
 }
 
 var saveSnapshotCompression = snapshotOptions.compressionDefault;
@@ -3538,7 +3539,7 @@
 function saveSnapshotModalInit() {
     $('#saveSnapshotModalProgressSection').hide();
     $('#saveSnapshotResolutionRadio').show();
-    saveSnapshotModalLog('info', 'Select resolution and click <b>Save</b>');
+    saveSnapshotModalLog('info', '选择分辨率，然后单击 <b>保存</b>');
     $('#saveSnapshotExport').removeClass('disabled');
 
     loadBootstrapSlider(function () {
@@ -3604,12 +3605,12 @@
                 saveSnapshotSelectedSecondsPerPoint = value;
                 saveSnapshotModalShowExpectedSize();
 
-                var seconds = ' seconds ';
+                var seconds = ' 秒 ';
                 if (value === 1) {
-                    seconds = ' second ';
+                    seconds = ' 秒 ';
                 }
 
-                return value + seconds + 'per point' + ((value === options.data.update_every) ? ', server default' : '').toString();
+                return value + seconds + '每点' + ((value === options.data.update_every) ? ', 服务器默认' : '').toString();
             }
         });
     });
@@ -3625,7 +3626,7 @@
 
             var filename = document.getElementById('saveSnapshotFilename').value;
             // console.log(filename);
-            saveSnapshotModalLog('info', 'Generating snapshot as <code>' + filename.toString() + '</code>');
+            saveSnapshotModalLog('info', '将快照生成为<code>' + filename.toString() + '</code>');
 
             var save_options = {
                 stop_updates_when_focus_is_lost: false,
@@ -3682,7 +3683,7 @@
             };
 
             if (typeof snapshotOptions.compressions[saveData.compression] === 'undefined') {
-                alert('unknown compression method: ' + saveData.compression);
+                alert('未知压缩方式：' + saveData.compression);
                 saveData.compression = 'none';
             }
 
@@ -3718,7 +3719,7 @@
             saveData.before_ms = NETDATA.globalPanAndZoom.force_before_ms;
             saveData.duration_ms = saveData.before_ms - saveData.after_ms;
             saveData.data_points = Math.round((saveData.before_ms - saveData.after_ms) / (saveSnapshotSelectedSecondsPerPoint * 1000));
-            saveSnapshotModalLog('info', 'Generating snapshot with ' + saveData.data_points.toString() + ' data points per dimension...');
+            saveSnapshotModalLog('info', '使用生成快照 ' + saveData.data_points.toString() + ' 每个维度的数据点...');
 
             var charts_count = 0;
             var charts_ok = 0;
@@ -3758,11 +3759,11 @@
             NETDATA.abortAllRefreshes();
 
             var size = 0;
-            var info = ' Resolution: <b>' + saveSnapshotSelectedSecondsPerPoint.toString() + ((saveSnapshotSelectedSecondsPerPoint === 1) ? ' second ' : ' seconds ').toString() + 'per point</b>.';
+            var info = ' 分辨率: <b>' + saveSnapshotSelectedSecondsPerPoint.toString() + ((saveSnapshotSelectedSecondsPerPoint === 1) ? ' second ' : ' seconds ').toString() + 'per point</b>.';
 
             function update_chart(idx) {
                 if (saveSnapshotStop === true) {
-                    saveSnapshotModalLog('info', 'Cancelled!');
+                    saveSnapshotModalLog('info', '取消!');
                     saveSnapshotRestore();
                     return;
                 }
@@ -3792,7 +3793,7 @@
                             state.log('failed to be updated: ' + reason);
                         }
 
-                        saveSnapshotModalLog((charts_failed) ? 'danger' : 'info', 'Generated snapshot data size <b>' + (Math.round(size * 100 / 1024 / 1024) / 100).toString() + ' MB</b>. ' + ((charts_failed) ? (charts_failed.toString() + ' charts have failed to be downloaded') : '').toString() + info);
+                        saveSnapshotModalLog((charts_failed) ? 'danger' : 'info', '生成的快照数据大小<b>' + (Math.round(size * 100 / 1024 / 1024) / 100).toString() + ' MB</b>. ' + ((charts_failed) ? (charts_failed.toString() + ' 图表下载失败') : '').toString() + info);
 
                         if (idx > 0) {
                             update_chart(idx);
@@ -3807,7 +3808,7 @@
                             saveObjectToClient(saveData, filename);
 
                             if (charts_failed > 0) {
-                                alert(charts_failed.toString() + ' failed to be downloaded');
+                                alert(charts_failed.toString() + ' 下载失败');
                             }
 
                             saveSnapshotRestore();
@@ -3870,7 +3871,7 @@
 
             document.getElementById('browser_timezone').innerText = NETDATA.options.browser_timezone;
             document.getElementById('server_timezone').innerText = NETDATA.options.server_timezone;
-            document.getElementById('current_timezone').innerText = (NETDATA.options.current.timezone === 'default') ? 'unset, using browser default' : NETDATA.options.current.timezone;
+            document.getElementById('current_timezone').innerText = (NETDATA.options.current.timezone === 'default') ? '未设置，使用浏览器默认' : NETDATA.options.current.timezone;
 
             if (self.prop('checked') === NETDATA.dateTime.using_timezone) {
                 self.bootstrapToggle(NETDATA.dateTime.using_timezone ? 'off' : 'on');
@@ -4033,7 +4034,7 @@
         scrollDashboardTo();
 
         // restore the scrollspy at the proper position
-        $(document.body).scrollspy('refresh');
+        $(document.body).scrollspy('刷新');
         $(document.body).scrollspy('process');
 
         return;
@@ -4246,7 +4247,7 @@
 
     $('#updateModal')
         .on('show.bs.modal', function () {
-            versionLog('checking, please wait...');
+            versionLog('检查中，请稍候...');
         })
         .on('shown.bs.modal', function () {
             notifyForUpdate(true);
@@ -4656,7 +4657,7 @@
         }
     ).then((response) => {
         if (!response.ok) {
-            throw Error("Cannot fetch known accounts");
+            throw Error("无法提取已知帐户");
         }
         return response.json();
     }).then((payload) => {
@@ -5067,7 +5068,7 @@
     }
 
     if (agentsToSync) {
-        console.log("Synchronizing with netdata.cloud.");
+        console.log("与netdata.cloud同步。");
 
         postCloudAccountAgents(agentsToSync).then((agents) => {
             // TODO: clear syncTime on error!
